src/sentry/api/endpoints/project_performance_issue_settings.py

```diff
@@ -115,5 +140,14 @@ def delete(self, request: Request, project) -> Response:
         if not self.has_feature(project, request):
             return self.respond(status=status.HTTP_404_NOT_FOUND)
 
-        project.delete_option(SETTINGS_PROJECT_OPTION_KEY)
+        project_settings = project.get_option(SETTINGS_PROJECT_OPTION_KEY, default={})
+
+        if project_settings:
+            settings_only_with_admin_options = {
+                option: project_settings[option]
+                for option in project_settings
+                if option in super_admin_project_settings_options
+            }
+            project.update_option(SETTINGS_PROJECT_OPTION_KEY, settings_only_with_admin_options)
```

Comment:
If we find out that this is only just admin options and they're all set to true, I think we could also just delete the option here

Reply:
Just to update on my comment, it shouldn't be if they're all `True` but rather if we can compare them against their default (since some performance issues might be in development and not turned on)

Reply:
We talked about it @narsaynorath, it's not really a necessity here.